FinniversKit/Sources/Components/Ribbon/RibbonViewRepresentable.swift

```diff
@@ -0,0 +1,18 @@
+import SwiftUI
+
+public struct RibbonViewRepresentable: UIViewRepresentable {
+    let ribbonViewModel: RibbonViewModel
+    
+    public init?(ribbonViewModel: RibbonViewModel?) {
+        guard let ribbonViewModel else { return nil }
+        self.ribbonViewModel = ribbonViewModel
+    }
+    
+    public func makeUIView(context: Context) -> RibbonView {
+        let view = RibbonView(viewModel: ribbonViewModel)
+        return view
+    }
+    
+    public func updateUIView(_ uiView: RibbonView, context: Context) {
```

Comment:
While conforming to any Protocols try to avoid braking of SOLID's Interface Segregation Principle principle if it possible (clients should not be forced to implement interfaces they don’t use)

Reply:
noted